bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileOrRendition.java

```diff
@@ -340,13 +339,10 @@ public InputStream getStream() throws IOException {
                 return currentStream;
 
             } catch (URISyntaxException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
                 throw new IOException("Bad URI format", ex);
             } catch (JSchException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
                 throw new IOException("Error with connection", ex);
             } catch (SftpException ex) {
-                Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.  Errors in this class don't always bubble up where they can be inspected easily.